src/TSMapEditor/Initialization/MapLoader.cs

```diff
@@ -1165,7 +1218,9 @@ public static void ReadHouses(IMap map, IniFile mapIni)
                     if (houseType == null)
                     {
                         houseType = map.GetHouseTypes()[0];
-                        AddMapLoadError($"Nonexistent Country= or no Country= specified for House {houseName}. This makes it default to the first standard Country ({houseType.ININame}).");
+                        AddMapLoadError(string.Format(Translate("MapLoader.ReadHouses.InvalidHouseType", 
```

Comment:
Might be best to use "Country" here - this is an error message for RA2/YR which also uses Country as a technical term.

```suggestion
                        AddMapLoadError(string.Format(Translate("MapLoader.ReadHouses.InvalidCountry", 
```